store/testdata/test_deal.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Where this test data will be used for?

Reply:
Actually, this file was used in `TestIpfsGet()` which is located in `ipfs_test.go`.
This is now useless, so I'll remove it.

Reply:
Also, I'll delete `testdata` directory. This is now useless.